mod_conf_sql.html

```diff
@@ -195,6 +213,11 @@
   proftpd -c sql://foo:bar@localhost/db:proftpd/ctxt:ftpctxt/conf:ftpconf/map:ftpmap/base_id=<i>baseid</i>
 </pre>
 
+<p> for sqlite dbuser and dbpass are note used, use dbserver for file path and use \ in place of / for the file path : </p>
+<pre>
+  proftpd -c sql://nouser:nopass@\path\to\sqlite.db/db:sqlite/ctxt:ftpctxt/conf:ftpconf/map:ftpmap
```

Comment:
I'm wondering if we can find some other way of providing this path, rather than using backslashes; this interface seems a bit awkward.